### PR TITLE
Add Size Command

### PR DESCRIPTION
**Pull request by vahid-sohrabloo**

### CODE REVIEW

**Comment by jlaffaye** [comment]
Can you add a test for this new function ?

**Comment by us3r64** [comment]
For consistency size should be int64

```go
// Size issues a SIZE FTP command.
func (c *ServerConn) FileSize(path string) (int64, error) {
	_, message, err := c.cmd(StatusFile, "SIZE %s", path)
	if err != nil {
		return 0, err
	}

	size, err := strconv.ParseInt(message, 10, 64)
	if err != nil {
		return 0, err
	}

	return size, nil
}
```

**Comment by vahid-sohrabloo** [comment]
I added

**Comment by vahid-sohrabloo** [comment]
I added

File: ftp.go [hunk 1]
```diff
@@ -407,6 +407,21 @@ func (c *ServerConn) CurrentDir() (string, error) {
...
 }
 
+// FileSize issues a SIZE FTP command, which Returns the size of the file
+func (c *ServerConn) FileSize(path string) (int64, error) {
+	_, msg, err := c.cmd(StatusFile, "SIZE %s", path)
+	if err != nil {
+		return 0, err
+	}
+
+	size, err := strconv.ParseInt(msg, 10, 64)
```
Comment (corny) [inline ftp.go]: just do `return strconv.ParseInt(msg, 10, 64)`

File: client_test.go [hunk 2]
```diff
@@ -96,6 +96,19 @@ func testConn(t *testing.T, disableEPSV bool) {
...
 
+	fileSize, err := c.FileSize("tset")
+	if err != nil {
+		t.Error(err)
+	}
+	if fileSize != 14 {
+		t.Errorf("file size %q, expected %q", fileSize, 14)
+	}
+
+	fileSize, err = c.FileSize("not-found")
```
Comment (corny) [inline client_test.go]: Please also test the `fileSize` being zero.

**Comment by jlaffaye** [comment]
THanks!
